client/data/home/use-home-layout-query-params.ts

```diff
@@ -8,7 +9,8 @@ export interface HomeLayoutQueryParams {
 }
 
 export function useHomeLayoutQueryParams(): HomeLayoutQueryParams {
-	const { dev, view } = useSelector( getCurrentQueryArguments ) as any;
+	const dev = get( useSelector( getCurrentQueryArguments ), 'dev', undefined );
+	const view = get( useSelector( getCurrentQueryArguments ), 'view', undefined );
```

Comment:
This doesn't look like an improvement. Just `const { dev, view } = useSelector( ... )` should work. If it doesn't, there's something wrong with the types and they should be fixed. Calling the selector twice and introducing a new usage of Lodash is not good.